gdzig/object.zig

```diff
@@ -189,12 +189,12 @@ pub const PropertyInfo = extern struct {
         const name = try allocator.create(StringName);
         name.* = StringName.fromComptimeLatin1(field_name);
 
+        // TODO: this causes a panic
         const hint_string = if (opt.hint_string.len > 0) blk: {
-            // TODO: this causes a panic
             const hint_string = try allocator.create(String);
             hint_string.* = String.fromLatin1(opt.hint_string);
             break :blk hint_string;
-        } else null;
+        } else null; // WARNING: this null causes the crash
```

Comment:
Perhaps we can use an empty string as the default. Unfortunately I don't think we can create a comptime empty `String`. It's technically possible but not practical.